tests/publiccloud/create_aistack_env.pm

```diff
@@ -0,0 +1,247 @@
+    my $rke2_url = get_var('RKE2_URL');
+    my $kubectl_url = get_var('KUBECTL_URL');
+    my $helm_url = get_var('HELM_URL');
+
+    record_info('Dep pkg install');
+    trup_call("pkg install curl git docker");
+    process_reboot(trigger => 1);
+    systemctl("enable docker");
+    systemctl("start docker");
+    systemctl("status docker");
+    script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
+    script_run("sh ./install_rke2.sh");
+    script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
+    systemctl("enable rke2-server.service");
+    systemctl("start rke2-server.service");
+    systemctl("status rke2-server.service");
+    script_run("rke2 --version");
+    script_run("curl -sSL $helm_url -o ./install_helm.sh && chmod 775 ./install_helm.sh");
+    script_run("sh ./install_helm.sh");
+    script_run("helm version");
+    script_run("curl -sSL $kubectl_url -o ./kubectl && chmod +x ./kubectl");
+    script_run("sudo mv ./kubectl /usr/local/bin/");
+    script_run("kubectl version --client");
+}
+
+sub install_dependency_components {
+    my ($instance) = @_;
+    my $cert_repo = get_var('HELM_CERTS');
+    my $ingress_repo = get_var('HELM_INGRESS');
+    my $ing_ver = get_var('ING_VERSION');
+    # Add Ingress Controller to open-webui endpoint
+    assert_script_run("helm repo add $ingress_repo");
+    assert_script_run("helm repo update");
+    assert_script_run("helm upgrade --install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --set controller.service.type=ClusterIP --version $ing_ver --create-namespace", timeout => 120);
+
+    # Add cert-manager repo,install
+    assert_script_run("helm repo add $cert_repo");
+    assert_script_run("helm repo update");
+    assert_script_run("helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.15.2 --set crds.enabled=true", timeout => 120);
+}
+
+sub config_kubectl {
+    my ($instance) = @_;
+    # config kubectl
+    assert_script_run("mkdir -p ~/.kube");
+    assert_script_run("sudo cp /etc/rancher/rke2/rke2.yaml ~/.kube/config");
+    assert_script_run("kubectl config get-contexts");
+    assert_script_run("kubectl config use-context default");
+    assert_script_run("kubectl config view");
+}
+
+sub install_aistack_chart {
+    my ($instance, $ai_chart_repo, $namespace, $vf_name) = @_;
+    my $SECRET_application_collection = get_var('_SECRET_DOCKER');
+    my $cert_repo = get_var('HELM_CERTS');
+    my $ingress_repo = get_var('HELM_INGRESS');
+    my $docker_user_name = 'yarunachalam@suse.com';
+    #my $repo_url = 'github.com/SUSE/private-ai-charts';
+    my $repo_url = get_var('HELM_CHARTS');
+    my $git_token = get_var('_SECRET_SSH');
+    my $ing_ver = get_var('ING_VERSION');
+    #my $ing_ver = '4.11.1';
+    my $local_path_url = 'https://raw.githubusercontent.com/yarunachalam/os-autoinst-distri-opensuse/aistack_basic/data/aistack/local_path_storage.yaml';
```

Comment:
This **path** shall point to `os-autoinst` user repo, not to a forked one.  
Moreover, you should use the standart url `https://github.com/...`

Reply:
for now I have in the fork only..Once it's merged will have the os-auto-inst repo. Let me have the url commented

Reply:
I have replaced all the data/ <values> references to data_url ..Currently comment those lines. Once PR merged. I will have those files in data dir. I will uncomment these lines
`    #assert_script_run( "curl " . data_url("aistack/$local_storage_name") . " -o $local_storage_name", 60);

   #assert_script_run( "curl " . data_url("aistack/$vf_name") . " -o $vf_name", 60);


  #assert_script_run( "curl " . data_url("aistack/$file_name") . " -o $file_name", 60);

`

Reply:
ok, temporary w.a., to be fixed as soon data are ready.